internal/operators/osc/manifest.go

```diff
@@ -0,0 +1,101 @@
+package osc
+
+import (
+	"bytes"
+	"html/template"
+)
+
+func Manifests() (map[string][]byte, []byte, error) {
+
+	oscNamespaceManifest, err := getNamespace(Namespace)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	oscOperatorGroupManifest, err := getOperatorGroup(Namespace)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	oscSubscriptionManifest, err := getSubscription(Namespace, SubscriptionName, Source, SourceName)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	openshiftManifests := make(map[string][]byte)
+
+	openshiftManifests["50_openshift-osc_ns.yaml"] = oscNamespaceManifest
+	openshiftManifests["50_openshift-osc_operator_group.yaml"] = oscOperatorGroupManifest
+	openshiftManifests["50_openshift-osc_subscription.yaml"] = oscSubscriptionManifest
+
+	return openshiftManifests, nil, err
+}
+
+func executeTemplate(data map[string]string, contentName, content string) ([]byte, error) {
+	tmpl, err := template.New(contentName).Parse(content)
+	if err != nil {
+		return nil, err
+	}
+	buf := &bytes.Buffer{}
+	err = tmpl.Execute(buf, data)
+	if err != nil {
+		return nil, err
+	}
+	return buf.Bytes(), nil
+}
+
+func getSubscription(namespace, subscription, source, sourceName string) ([]byte, error) {
+	data := map[string]string{
+		"OPERATOR_NAMESPACE":         namespace,
+		"OPERATOR_SUBSCRIPTION_NAME": subscription,
+		"OPERATOR_SOURCE":            source,
+		"OPERATOR_SOURCE_NAME":       sourceName,
+	}
+	return executeTemplate(data, "oscSubscriptionManifest", oscSubscriptionManifest)
+}
+
+func getNamespace(namespace string) ([]byte, error) {
+	data := map[string]string{
+		"OPERATOR_NAMESPACE": namespace,
+	}
+	return executeTemplate(data, "oscNamespaceManifest", oscNamespaceManifest)
+}
+
+func getOperatorGroup(namespace string) ([]byte, error) {
+	data := map[string]string{
+		"OPERATOR_NAMESPACE": namespace,
+	}
+	return executeTemplate(data, "oscOperatorGroupManifest", oscOperatorGroupManifest)
+}
+
+const oscNamespaceManifest = `
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{.OPERATOR_NAMESPACE}}
+`
+
+const oscOperatorGroupManifest = `
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: sandboxed-containers-operator-group
+  namespace: {{ .OPERATOR_NAMESPACE }}
+spec:
+  targetNamespaces:
+    - {{ .OPERATOR_NAMESPACE }}
+`
+
+const oscSubscriptionManifest = `
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: {{.OPERATOR_SUBSCRIPTION_NAME}}
+  namespace: {{.OPERATOR_NAMESPACE}}
+spec:
+  channel: stable
```

Comment:
there is no channel stable  -- you can remove this to use the default channel instead 

that's the reason the job is failing can't find the channel 

```bash
$ oc-mirror list operators  --catalog=registry.redhat.io/redhat/redhat-operator-index:v4.17 --package container-security-operator
NAME                         DISPLAY NAME  DEFAULT CHANNEL
container-security-operator                stable-3.13

PACKAGE                      CHANNEL      HEAD
container-security-operator  quay-v3.4    container-security-operator.v3.4.7
container-security-operator  quay-v3.5    container-security-operator.v3.5.7
container-security-operator  stable-3.10  container-security-operator.v3.10.6
container-security-operator  stable-3.11  container-security-operator.v3.11.6
container-security-operator  stable-3.12  container-security-operator.v3.12.5
container-security-operator  stable-3.13  container-security-operator.v3.13.1
container-security-operator  stable-3.6   container-security-operator.v3.6.10
container-security-operator  stable-3.7   container-security-operator.v3.7.14
container-security-operator  stable-3.8   container-security-operator.v3.8.15
container-security-operator  stable-3.9   container-security-operator.v3.9.8
```